R/api-functions.R

```diff
@@ -0,0 +1,79 @@
+# Data package identifier reservation ------------------------------------------
+#' Reserve EDI ID 
```

Comment:
change title to "Reserve EDI Data Package Identifier"

Reply:
done